shared/common-adapters/name-with-icon/container.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import {compose, connect, setDisplayName} from '../../util/container'
+import * as Route from '../../actions/route-tree'
+import {teamsTab} from '../../constants/tabs'
+import * as ProfileGen from '../../actions/profile-gen'
+import * as TrackerGen from '../../actions/tracker-gen'
+import NameWithIcon, {type NameWithIconProps} from '.'
+import {isMobile} from '../../constants/platform'
+
+export type ConnectedNameWithIconProps = {|
+  ...NameWithIconProps,
+  onClick?: ((SyntheticEvent<> | void) => void) | 'tracker' | 'profile' | 'trackerProfileFallback',
```

Comment:
do we still need to be able to have `onClick` be an arbitrary function? Seems like we want to disallow that, and users that need it can just use plain `NameWithIconProps`

Reply:
Nope! I had kinda originally envisioned ConnectedNameWithIcon to totally replace NameWithIcon but since they currently are both usable in their current forms I see no reason to keep a function definition in ConnectedNameWithIconProps.